### ISSUE TITLE
Flameshot is not working on Fedora OS 40 and 41.

### ISSUE THREAD

**Issue by IgaoWolf** [flameshot-org/flameshot#3770]
### Flameshot Version

Não está funcionando o flameshot no SO Fedora 40 e 41

### Installation Type

Linux, MacOS, or Windows Package manager (apt, pacman, eopkg, choco, brew, ...), User repository (AUR), Flatpak from Flathub, Flatpak from Github, Snap from Snapcraft, Snap from, AppImage, Using the ready-made package from Github Releases, Compiled from source

### Operating System type and version

Fedora 40 e 41

### Workaround to Run Flameshot on Fedora 40 and Fedora 41

Follow these steps to install and set up Flameshot with a custom shortcut in Fedora 40 and Fedora 41.

1. **Download Flameshot AppImage:**
   - Go to [Flameshot Download Page](https://flameshot.org/#download).
   - Select **Linux** and download the AppImage version.

2. **Set Up Flameshot Directory:**
   - Open your terminal and create a dedicated directory for Flameshot:
     ```bash
     sudo mkdir --parents ~/Applications/Flameshot
     ```

3. **Move the AppImage to the Directory:**
   - Copy the downloaded AppImage to the newly created directory:
     ```bash
     sudo cp /home/<PASTA>/Downloads/Flameshot-12.1.0.x86_64.AppImage ~/Applications/Flameshot
     ```

4. **Make the AppImage Executable:**
   - Grant execute permissions to the AppImage file:
     ```bash
     sudo chmod +x ~/Applications/Flameshot/Flameshot-*.x86_64.AppImage
     ```

5. **Create a Script for GUI Launching:**
   - Open a new file named `flameshot_gui.sh` in the `Applications` directory to create a launch script:
     ```bash
     sudo vim ~/Applications/flameshot_gui.sh
     ```
   - Add the following content to the script file:
     ```bash
     #!/bin/bash
     /home/<PASTA>/Applications/Flameshot/Flameshot-*.x86_64.AppImage gui
     ```
   - Save and exit, then make the script executable:
     ```bash
     sudo chmod +x ~/Applications/flameshot_gui.sh
     ```

6. **Adjust Keyboard Shortcut Settings:**
   - Go to **Settings** > **Keyboard** > **View and Customize Shortcuts** > **Screenshots**.
   - **Disable all options except for Screen Recorder**.

7. **Create a Custom Shortcut for Flameshot:**
   - In **Settings** > **Keyboard** > **View and Customize Shortcuts** > **Custom Shortcuts**:
     - Create a new entry:
       - **Name**: Flameshot GUI
       - **Command**: `/home/<PASTA>/Applications/flameshot_gui.sh`
     - Set the shortcut to `fn + Insert`, and ensure **PRINT** is shown as the active key for the shortcut.

After completing these steps, pressing **PRINT** (or `fn + Insert`) should launch Flameshot, providing the desired screenshot functionality on Fedora 40 and 41.

### System Information

NAME="Fedora Linux"
VERSION="41 (Workstation Edition)"

**Comment by mmahmoudian** [comment]
I don't believe your issue is Fedora related, and I believe it is Gnome Wayland. But because you didn't provide enough information in the "System Information" section, I cannot be sure. Please edit your original post and update that for transparency.

I also think that Flatpak would achieve the same results as AppImage. Have you considered trying that?

Also, I wonder if this is a vug report or suggestion for documentation. Because you filled the bug report form, but I don't see much of the bug report details.

**Comment by IgaoWolf** [comment]
We have identified a recurring issue when attempting to use the tool via Flatpak, as, despite multiple attempts, it was not possible to make it work properly. The only solution we found was to use the AppImage version, which worked as expected. In light of this, I am updating the operating system information in this ticket. This incident qualifies as a root bug, preventing the correct use of the tool.

As a temporary measure, I have implemented a workaround that allows usage on both Fedora 40 and Fedora 41. It is worth noting that all team members using these Fedora versions are encountering the same issue. Any updates or a definitive solution would be greatly appreciated to mitigate this situation for everyone.

**Comment by Chevek** [comment]
What if you launch flameshot (rpm from default repository) from the command line? Here it works that way!

**Comment by IgaoWolf** [comment]
Now, it's working fine using the AppImage, but have issue for utilizing this.

**Comment by raphamzn** [comment]
@IgaoWolf boa Igão, funcionou aqui.

**Comment by alzanlorensi** [comment]
Vlw cara ajudou aqui

**Comment by abdelouahabb** [comment]
Landed here from this issue, and it seems mine was this :

```
→ journalctl -b | grep flameshot
mai 19 15:16:51 alien-popos flameshot.desktop[7208]: /usr/local/bin/flameshot: /lib/x86_64-linux-gnu/libstdc++.so.6: version `CXXABI_1.3.15' not found (required by /usr/local/bin/flameshot)

```

Which means Pop OS 22.04 is not compatible if you try to compile it by yourself using a recent GCC
